untitled.js

```diff
@@ -0,0 +1,14 @@
+var name = 'Christian'
+let age = 15
+const gender = 'male' 
+let grade = 9;
+let height = 5,9;
+let weight = 155;
+const bday = 3/12;
+let todayismonday = {false;}
+const Christian = {name:Christian , favoritefood:'dominos'}
```

Comment:
The value for the `name` property must be enclosed in quotes since it's a string.